audbackend/core/api.py

```diff
@@ -63,6 +67,14 @@ def access(
     on the backend with alias ``name``
     (see :func:`audbackend.register`).
 
+    .. Warning::
+
+        ``audbackend.access()`` is deprecated
+        and will be removed in version 2.2.0.
+        Repositories on backends are instead accessed
+        by instantiating the corresponding backend class,
+        e.g. :meth:`audbackend.backend.FileSystem(host, repo)`.
```

Comment:
Doesn't look right to me:

![image](https://github.com/audeering/audbackend/assets/10383417/667b2df0-4aaa-4c69-847e-a67db4f95775)

One could also argue that the replacement for `access` is `open`.

Reply:
I updated the warning message to

![image](https://github.com/audeering/audbackend/assets/173624/3ec2e5ec-1c4a-4315-8e58-81c67d6e23dd)